src/azure-cli-core/azure/cli/core/breaking_change.py

```diff
@@ -0,0 +1,282 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import abc
+
+NEXT_BREAKING_CHANGE_RELEASE = '2.67.0'
```

Comment:
Can this variable be configured in the ADO pipeline where the breaking change doc is published?

Reply:
This is used when diaplaying the breaking change message that targets at the next breaking change release. I plan to create an ado task to create PRs to update this line before each breaking change window.